api/v1alpha1/etcd.go

```diff
@@ -243,6 +243,10 @@ type ClientService struct {
 	// Labels specify the labels that should be added to the client service
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
+	// TrafficDistribution defines the traffic distribution preference that should be added to the client service.
+	// More info: https://kubernetes.io/docs/reference/networking/virtual-ips/#traffic-distribution
+	// +optional
+	TrafficDistribution *string `json:"trafficDistribution,omitempty"`
```

Comment:
Update  [sample configuration file](config/samples/) , add a placeholder for the TrafficDistribution field to guide users